ztp/tools/siteconfig-converter/README.md

```diff
@@ -77,15 +79,73 @@ make build
   siteconfig.yaml
 ```
 
-#### Creating custom templates 
+#### Live Cluster Migration
 
-Refer to siteconfig [docs](https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.13/html/multicluster_engine_operator_with_red_hat_advanced_cluster_management/siteconfig-intro#create-custom-templates) on how to create custom templates. 
+When migrating a live cluster, the manifests generated by ClusterInstance must exactly match those already deployed. However, due to API deprecation, ClusterInstance cannot replicate older versions of `AgentClusterInstall` resources that use deprecated fields. Specifically, this affects clusters with the following deprecated fields: `apiVIP`, `ingressVIP`, and `manifestsConfigMapRef`.
 
-#### Creating HostFirmWare custom template instead of biosConfigRef
+If you attempt to live migrate a cluster with any of these deprecated fields, you will encounter an error similar to this:
 
-Create a configmap containing your BIOS config: 
+```
+message: "failed to update resource during patch operation: admission webhook
+\"agentclusterinstallvalidators.admission.agentinstall.openshift.io\" denied
+the request: Attempted to change AgentClusterInstall.Spec which is immutable
+after install started, except for ClusterMetadata,IgnitionEndpoint fields.
+Unsupported change: \n\tManifestsConfigMapRefs: ([] => [{Name:sno1}])"
+```
+
+**Before performing the migration**, verify whether the existing `AgentClusterInstall` resource on the hub cluster contains any of these deprecated fields. If it does, you have two resolution options:
+
+**Option 1: Create a Custom Cluster Template**
+
+Create a custom cluster template that uses the deprecated singular field forms instead of the current plural forms. The following diff shows the differences between the original template and the custom template:
```

Comment:
```suggestion
Create a ConfigMap that contains the custom cluster template that uses the deprecated singular field forms instead of the current plural forms. The following diff shows the differences between the original template and the custom template:
```?
It's initially unclear where the custom template should be put.